packages/bun-types/package.json

```diff
@@ -1,25 +1,37 @@
 {
   "name": "bun-types",
-  "repository": "https://github.com/oven-sh/bun",
```

Comment:
Does repo need to go?

Reply:
Its not getting deleted, it just got moved down a bit and its using the object form of it for better redirects in npm